bin/bit.js

```diff
@@ -64,8 +71,30 @@ function updateOrLaunch(updateCommand) {
 function loadCli() {
   return require('../dist/app.js');
 }
-
+function promptAnalyticsIfNeeded(cb) {
+  const { analyticsPrompt, errorReportingPrompt } = require('../dist/prompts');
+  const cmd = process.argv.slice(2);
+  if (cmd.length && cmd[0] !== 'config') {
+    const analytics = getSync(constants.CFG_ANALYTICS_REPORTING_KEY);
+    const error_reporting = getSync(constants.CFG_ANALYTICS_ERROR_REPORTS_KEY);
+    const analyticsAnswer = !R.isNil(analytics) ? yn(analytics, { default: false }) : undefined;
+    const errorsAnswer = !R.isNil(error_reporting) ? yn(error_reporting, { default: false }) : undefined;
+    if (R.isNil(analyticsAnswer) && R.isNil(errorsAnswer)) {
+      const uniqId = uniqid();
+      setSync(constants.CFG_ANALYTICS_USERID_KEY, uniqId);
```

Comment:
put this inside an if to check if it exists.
If a user deleted the analytics/errors value but didn't delete this, I prefer to keep this value as before